rotkehlchen/accounting/cost_basis/base.py

```diff
@@ -388,14 +395,14 @@ def calculate_spend_cost_basis(
         Calculates the cost basis of the spend using the average cost basis method.
 
         Formula is taken from:
-        https://www.adjustedcostbase.ca/blog/how-to-calculate-adjusted-cost-base-acb-and-capital-gains/
+        https://www.canada.ca/content/dam/cra-arc/formspubs/pub/t4037/t4037-22e.pdf, page 23
```

Comment:
Keep both links. I would say the canada.ca link is ofcourse the "official one" but I think the blog at adjustedcostbase.ca is best to understand.

But perhaps as I said above put that in the class docstring with the links I added in the issue for both jurisdictions and explain it applies to both. Also link to the issue comment itself. It's very detailed and helps explain everything, so it should be linked in the class: https://github.com/rotki/rotki/issues/5561#issuecomment-1423338938